sdk/containerregistry/Azure.Containers.ContainerRegistry/src/ContainerRegistryClient.cs

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Specialized;
+using System.Threading;
+using System.Web;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.Containers.ContainerRegistry
+{
+    /// <summary> The Repository service client. </summary>
+    public partial class ContainerRegistryClient
+    {
+        private readonly Uri _endpoint;
+        private readonly HttpPipeline _pipeline;
+        private readonly ClientDiagnostics _clientDiagnostics;
+        private readonly RepositoryRestClient _restClient;
+
+        /// <summary>
+        /// <paramref name="endpoint"/>
+        /// </summary>
+        public ContainerRegistryClient(Uri endpoint, string username, string password) : this(endpoint, username, password,  new ContainerRegistryClientOptions())
```

Comment:
These constructors will be changed to standard Track 2 constructors before first preview release, per: https://github.com/Azure/azure-sdk-for-net/issues/19469